.devcontainer/cpp/test/testsuite.bats

```diff
@@ -205,3 +205,15 @@ function build_and_run_with_sanitizers() {
   assert_failure
   assert_output --partial "runtime error: load of null pointer"
 }
+
+@test "using clang-uml to generate uml class diagram" {
+  run cmake --preset clang
+  assert_success
+
+  run cmake --build --preset clang-uml
+  assert_success
+
+  run clang-uml --config ./clang-uml/.clang-uml
+  assert_success
+  assert_output --partial "Written test_class diagram to"
```

Comment:
Should we extend this test any further? Do you want to know if the file actually exists? If it contains the proper content?

Reply:
Contents are not static, there seems to be a seed that generates random identifiers. So no I won't be verifying the output any further :)